### ISSUE TITLE
Disabled tag needs aria-disabled

### ISSUE THREAD

**Issue by yalagand** [adobe/spectrum-web-components#741]
Disabled tags currently have insufficient color contrast. Accessibility tooling ignores color contrast issues on disabled elements. 

The current markup has disabled attribute on it: <sp-tag disabled="" role="listitem" tabindex="-1">Tag 3</sp-tag>. But HTML disabled attribute is supported only by button, command, fieldset, keygen, optgroup, option, select, textarea and input.

In order to convey to screenreaders that the listitem is disabled, it needs aria-disabled='true'. If aria-disabled is added, accessibility tools won't raise a color contrast issue on the disabled tag and lisitem is also then properly conveyed as disabled.
